Helper/Entity/ProductHelper.php

```diff
@@ -201,10 +218,8 @@ public function getProductCollectionQuery(
         $onlyVisible = true,
         $includeNotVisibleIndividually = false
     ) {
-        /** @var \Magento\Catalog\Model\ResourceModel\Product\Collection $products */
-        $products = $this->objectManager->create('Magento\Catalog\Model\ResourceModel\Product\Collection');
-
-        $products = $products
+        $productCollection = $this->productCollectionFactory->create();
+        $products = $productCollection    
```

Comment:
```suggestion
        $products = $productCollection
```

Reply:
Hello @peterjaap !
Thanks for this PR and sorry that we didn't have time to handle it before.
Is it possible to submit this change so the codestyle check can pass ?
Thanks in advance ;)